Flaky tests test_e2e_14_mef_eline.py::test_005_create_evc_on_nni

From the last execs:

```
rerun: 0
tests/test_e2e_14_mef_eline.py::TestE2EMefEline::test_005_create_evc_on_nni: 2025-10-05,05:47:20.771359 - 2025-10-05,05:47:36.711949
self = <tests.test_e2e_14_mef_eline.TestE2EMefEline object at 0x7fd0b5331950>

   def test_005_create_evc_on_nni(self):
       """Test to evaluate how mef_eline will behave when the uni is actually
       an NNI."""
       api_url = KYTOS_API + '/kytos/mef_eline/v2/evc/'
       evc1 = self.create_evc(uni_a='00:00:00:00:00:00:00:16:5',
                              uni_z='00:00:00:00:00:00:00:11:1',
                              vlan_id=100,
                              max_paths=10)

       time.sleep(10)

       # It verifies EVC's data
       response = requests.get(api_url + evc1)
       data = response.json()
       assert data['enabled'] == True
       assert data['active'] == True

       trace_id = self.do_sdntrace('00:00:00:00:00:00:00:16', 5, 100)
       result = self.wait_sdntrace_result(trace_id)

     assert len(result) == 7
E       AssertionError: assert 3 == 7
E        +  where 3 = len([{'dpid': '00:00:00:00:00:00:00:16', 'port': 5, 'time': '2025-10-05 05:47:33.239710', 'type': 'starting'}, {'dpid': '0...time': '0:00:00.545795', 'type': 'trace'}, {'msg': 'none', 'reason': 'done', 'time': '0:00:03.065435', 'type': 'last'}])

tests/test_e2e_14_mef_eline.py:156: AssertionError

```

--- a/tests/test_e2e_14_mef_eline.py
+++ b/tests/test_e2e_14_mef_eline.py
@@ -92,7 +92,8 @@ class TestE2EMefEline:
         vlan_id=100,
         primary_path=None,
         backup_path=None,
-        timeout=5,
+        max_retries=5,
+        wait_time=1,
         **kwargs,
     ):
         payload = {
@@ -116,16 +117,16 @@ class TestE2EMefEline:
         if kwargs:
             payload.update(kwargs)
         times = 0
-        while times < timeout:
+        while times < max_retries:
             api_url = KYTOS_API + '/kytos/mef_eline/v2/evc/'
             response = requests.post(api_url, json=payload)
             data = response.json()
             assert response.status_code == 201, response.text
             if data["deployed"] is True:
                 break
-            time.sleep(1)
+            time.sleep(wait_time)
             self.delete_evc(data["circuit_id"])
-            time.sleep(1)
+            time.sleep(wait_time)
             times += 1
         else:
             msg = "Time out to create EVC which deploys"

